microsoft-edge/extensions-chromium/developer-guide/api-support.md

```diff
@@ -74,6 +107,11 @@ Microsoft Edge doesn't support the following Extension APIs:
 * `chrome.identity.getAuthToken` - As an alternate, you can use `launchWebAuthFlow` to fetch an OAuth2 token to authenticate users.
 * `chrome.instanceID`
 * `chrome.readingList`<!-- [Chrome Extension API Proposal: chrome.readingList](https://docs.google.com/document/d/1f1wW2955nY5Rp3vfS0JGuo1kACOAmsjZupvLYknWKII/edit#heading=h.mbf7udvanbrr) -->
+<!--
```

Comment:
why did i list gcm & instanceID & readingList APIs here, hidden, instead of in main table?  URLs imply they are MV3.  Make sure main table lists all MV3 APIs.

gcm - listed in MV3 page?
instanceID - listed in MV3 page?
readingList - listed in MV3 page?

MV2 Extensions API Ref page:
https://developer.chrome.com/docs/extensions/mv2/reference/
MV3 Extensions API Ref page:
https://developer.chrome.com/docs/extensions/reference/api/

Reply:
cleaned up